src/viewer/Viewer.jsx

```diff
@@ -143,7 +142,7 @@ export default class Viewer extends Component {
       case 'video':
         return isMobile() ? NoViewer : VideoViewer
       case 'pdf':
-        return isMobileApp() ? NativePdfViewer : PdfViewer
```

Comment:
So we can remove `NativePdfViewer` file? And remove the native plugin in config.xml ?

Reply:
The file is removed, but yes, I forgot about the plugin!